Polygon_mesh_processing/test/Polygon_mesh_processing/pmp_compute_normals_test.cpp

```diff
@@ -98,8 +98,11 @@ void test(const Mesh& mesh,
     // tests on non triangular meshes are @todo
     if(CGAL::is_triangle(halfedge(f, mesh), mesh))
     {
-      if(PMP::is_degenerate_triangle_face(f, mesh))
-        assert(get(fnormals, f) == CGAL::NULL_VECTOR);
+      if (PMP::is_degenerate_triangle_face(f, mesh))
+      {
+        if (std::is_same<K, EPECK>())
```

Comment:
Just a note...

That is the first time is see a use of the call operator ofor a type traits: `std::is_same<K, EPECK>()`. So far I had not know that existed. [According to the documentation](https://en.cppreference.com/w/cpp/types/is_same), it is available from C++14 and is the same as:
  - `std::is_same<K, EPECK>::value`
  - or `std::is_same_v<K, EPECK>` (C++17).

The call operator `()` is `constexpr`, so the usages are equivalent.